src/accelerate/accelerator.py

```diff
@@ -158,7 +159,16 @@ class Accelerator:
             If `"all"` is selected, will pick up all available trackers in the environment and initialize them. Can
             also accept implementations of `GeneralTracker` for custom trackers, and can be combined with `"all"`.
         logging_dir (`str`, `os.PathLike`, *optional*):
-            A path to a directory for storing logs of locally-compatible loggers.
+            A path to a directory for storing logs of locally-compatible loggers. If not passed will save in
+            `project_dir` by default.
+        project_dir (`str`, `os.PathLike`, *optional*):
+            A path to a directory for storing data such as logs of locally-compatible loggers and potentially saved
+            checkpoints.
+        automatic_checkpoint_naming (`bool`, *optional*, defaults to `False`):
+            Whether saved states should be stored in `project_location` and be automatically iteratively named.
+        save_total_limit (`int`, *optional*):
+            The maximum number of checkpoints to keep if performing `automatic_checkpoint_naming`, will default to all
+            of them.
```

Comment:
This is starting to make too many arguments.

Reply:
Fixed with a `SaveConfiguration` class